test/instrumentation/modules/aws-sdk/dynamodb.js

```diff
@@ -0,0 +1,267 @@
+const agent = require('../../../..').start({
+  serviceName: 'test',
+  secretToken: 'test',
+  captureExceptions: false,
```

Comment:
nit: Perhaps also `cloudProvider: "none", logLevel: "off"` to remove noise from testing?

Reply:
✅